sources/packages/backend/apps/api/src/route-controllers/announcements/_tests_/e2e/announcement.controller.getAnnouncements.e2e-spec.ts

```diff
@@ -0,0 +1,106 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { Repository } from "typeorm";
+import { Announcement } from "@sims/sims-db";
+import { createTestingAppModule } from "../../../../testHelpers";
+import { addDays } from "@sims/utilities";
+
+describe("AnnouncementController(e2e)-getAnnouncements", () => {
+  let app: INestApplication;
+  let announcementsRepo: Repository<Announcement>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    announcementsRepo = dataSource.getRepository(Announcement);
+  });
+
+  it("Should return a saved announcement.", async () => {
```

Comment:
Suggestion: "Should return a saved currently active announcement."

Reply:
reworded